puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/foreman-x-develop-release.groovy

```diff
@@ -55,7 +55,7 @@ pipeline {
                         dir('foreman-packaging') {
                             obal(
                                 action: 'nightly',
-                                packages: project_name,
+                                packages: obal_package_name ?: project_name,
```

Comment:
This broke foreman-selinux: `groovy.lang.MissingPropertyException: No such property: obal_package_name for class: WorkflowScript`